stablehlo/tests/CheckOps.td

```diff
@@ -110,8 +110,8 @@ def CHECK_ExpectEqOp : Op<CHECK_Dialect, "expect_eq", [SameTypeOperands]> {
   }];
 
   let arguments = (ins
-      HLO_Tensor:$lhs,
-      HLO_Tensor:$rhs
+      HLO_TensorOrToken:$lhs,
+      HLO_TensorOrToken:$rhs
```

Comment:
Do we actually need to test tokens for equality? I don't think that the spec even defines what equality means for tokens - it just says that tokens are opaque values. Not doing this would also simplify quite a bit of code below.

Reply:
Sgtm